modules/jaeger-config-storage.adoc

```diff
@@ -189,18 +189,18 @@ spec:
 [id="jaeger-config-external-es_{context}"]
 == Connecting to an existing Elasticsearch instance
 
-Jaeger also allows you to use an existing (self-provisioned) Elasticsearch cluster for storage. You do this by specifying the URL of the existing cluster as the `spec:storage:options:es:server-urls` value in your configuration.
+Jaeger also allows you to use an already existing Elasticsearch cluster for storage (that is, an instance that was not auto-provisioned by the Jaeger Operator calling the Red Hat Elasticsearch Operator). You do this by specifying the URL of the existing cluster as the `spec:storage:options:es:server-urls` value in your configuration.
 
 .Restrictions
 
-* There can be only one Jaeger with self-provisioned Elasticsearch instance per namespace.
+* You should not share or reuse a {ProductName} logging Elasticsearch instance with Jaeger. The Elasticsearch cluster is meant to be dedicated for a single Jaeger instance.
```

Comment:
> ! should verb
Whenever possible, rewrite in a more direct way. Do not use "should" to refer to an action that must be performed. For example, when parameters must be reset, write "Reset the parameters" and not "You should reset the parameters".